pyproject.toml

```diff
@@ -33,6 +33,7 @@ classifiers = [
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
```

Comment:
I think the `python-version` should also be passed to the *qa-precommit-run.yml* workflow, so this line is not added. It would be nice to just have the PYTHON_VERSION in one place, not sure which is the best option, env vars?